test/Misc/Integer.jl

```diff
@@ -0,0 +1,18 @@
+@testset "Integers" begin
+    
+
+    @test is_commuatative(ZZ)==true
```

Comment:
This has a typo (did you test this test file?)

Reply:
I have fixed the typo professor. I think it works now!